nix/packages/default.nix

```
@@ -0,0 +1,12 @@
+{
+  pkgs ? import <nixpkgs> { },
+}:
```

```suggestion
{
  perSystem = 
    { pkgs, ... }:
    {
      packages = {
      ...
```
あとから色々とselfやsystemなどの引数を取りたくなることがあると思うので flake-parts のシステムに乗っかっておきたいです